docs/guides/debugging/clion.md

```diff
@@ -0,0 +1,51 @@
+# Debugging in CLion
+
+In this guide, we’ll be setting up IntelliJ CLion for debugging the [Core Process of your Tauri app].
+
+## Prerequisites
+
+You need to install the [IntelliJ Rust Plugin](https://plugins.jetbrains.com/plugin/8182-rust/docs) that Rust features are enabled.
+
+## Top-level Cargo Workspace
+
+By default Tauri places the Rust project in a subdirectory called `src-tauri`. CLion doesn't recognize Cargo projects that are not at the top-level like this, so we need to create a top-level Workspace that just points to our main `Cargo.toml` file: 
+
+```toml title=Cargo.toml
+[workspace]
+members = ["src-tauri"]
+```
+
+Before you proceeed, make sure that your project is fully loaded. If the indexing is finished and the Cargo tool window shows all the modules and targets of the workspace, you’re good to go.
```

Comment:
This is not true. Arguably it may be the easier solution but you can also tell CLion where the rust project is. For example if you open the main.rs file it should tell you that there is no Cargo project with a button saying something like "Attach project".

Reply:
Hmm, maybe nevermind. the debugger cries about missing files if we don't set the working directory to the src-tauri sub dir

Reply:
Yeah but even then, we I tested it I couldn't attach that nested folder at all. It just wouldn't recognise it

Reply:
you need to either select the src-tauri dir or the Cargo.toml file. It works 100% with the plugin, but for this use-case it'd require us the use src-tauri as the working dir for the Run command so idk if it's worth.